workbench/settings.py

```diff
@@ -97,13 +116,6 @@
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = '5ftdd9(@p)tg&amp;bqv$(^d!63psz9+g+_i5om_e%!32%po2_+%l7'
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
```

Comment:
I'd wager we don't need them in edx-platform either, the django docs mention these as the default options: https://docs.djangoproject.com/en/1.11/ref/templates/upgrading/#the-templates-settings

Unless you think we ought to be explicit about that?